packages/bento-design-system/src/Slider/Slider.tsx

```diff
@@ -108,18 +111,24 @@ type ThumbProps = {
   disabled?: boolean;
   showValue: boolean;
   autoFocus?: boolean;
+  onDragStatusChange?: (isDragging: boolean) => void;
 };
 
 function Thumb(props: ThumbProps) {
   const config = useBentoConfig().slider;
-  const { state, trackRef, index } = props;
+  const { state, trackRef, index, onDragStatusChange } = props;
   const inputRef = useRef<HTMLInputElement>(null);
   const {
     thumbProps: { style, ...thumbProps },
     inputProps,
+    isDragging,
   } = useSliderThumb({ index, trackRef, inputRef, isDisabled: props.disabled }, state);
   const { focusProps, isFocusVisible } = useFocusRing({ autoFocus: props.autoFocus });
 
+  useEffect(() => {
+    onDragStatusChange?.(isDragging);
+  }, [isDragging, onDragStatusChange]);
```

Comment:
I see from the [doc](https://react-spectrum.adobe.com/react-aria/useSlider.html) that the `useSlider` hook accepts  an `onChangeEnd` property described as follows:

> Fired when the slider stops moving, due to being let go.

Is this something that can be used to achieve a similar behaviour? Have you tried it?

Reply:
I have seen it but I also want to notify when the user starts dragging, so I think the current approach is more flexible

Reply:
Actually, I've changed my mind. I think it's cleaner to use `onChangeEnd`, which should be enough for the use case it originated this PR